about.html

```diff
@@ -0,0 +1,217 @@
+              </p>
+            </div>
+          </div>
+        </div>
+        <!-- <div class="container"> -->
+        <div id="contact" class="columns has-text-centered is-centered">
+          <div class="column is-5 mt-5 pt-5">
+            <p>
+              Feel free to contact us via Email or Phone<br />
+              ccgsummit@cryptolife.org
+            </p>
+          </div>
+        </div>
+        <!-- </div> -->
+      </section>
+    </main>
+    <!-- logo section -->
+    <div class="container">
+      <div class="columns">
+        <div class="column"></div>
+      </div>
+      <section>
+        <div id="ccgLogo" class="columns is-centered">
+          <div class="column is-7 has-text-centered">
+            <p>Crypto Currency Global Conference Logo 2023</p>
+          </div>
+        </div>
+        <div class="columns is-centered">
+          <div class="column is-7 has-text-centered">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin
+              ornare magna eros, quis feugiat facilisis.Lorem ipsum dolor sit
+              amet, consectetur adipiscing elit. Proin ornare magna eros, quis
+              feugiat facilisis.
+            </p>
+          </div>
+        </div>
+        <div class="columns is-centered pb-5 mb-5">
+          <div class="logo2">
+            <span class="black">CCG-</span>
+            <span class="red">CON</span>
+          </div>
+        </div>
+      </section>
+    </div>
+    <hr />
+    <section id="past">
+      <div class="columns is-centered">
+        <div class="column is-8 has-text-centered has-text-weight-bold">
+          <p>See the past CC Global Conferences</p>
+        </div>
+      </div>
+      <div id="takelook" class="columns is-centered">
+        <div class="column is-8 has-text-centered has-text-weight-bold">
+          <p>
+            Take a look at the last two CC Global Conferences in Washington and
+            Florida
+          </p>
+        </div>
+      </div>
+      <div class="columns is-centered">
+        <div class="column is-3">
+          <figure class="image is-large">
+            <img src="images/conference1.jpg" />
+            <div class="red-overlay"></div>
+          </figure>
+        </div>
+        <div class="column is-offset-1 is-3 ml-0">
+          <figure class="image is-large">
+            <img src="images/conference2.jpg" />
+            <div class="red-overlay"></div>
+          </figure>
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="foot2">
+        <div class="logo pt-3 pl-6">
+          <span class="black">CCG-</span>
+          <span class="red">CON</span>
+        </div>
+
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. <br />Proin
+          ornare magna eros,quis feugiat facilisis.
+        </p>
+      </div>
+    </footer>
+  </body>
+</html>
```

Comment:
- [ OPTIONAL ] I think the previous reviewer missed this but please know that it's crucial to fix this before submitting your project for the video Video Presentation Review :+1: 
- Currently, your mobile view for the `About page` doesn't look as it's supposed to. The entire page looks quite disorganized so kindly fix this and style the elements properly while ensuring the recommended spacing between them :+1: 
- Your Design

[Screencast from 08-10-2023 12:48:26 PM.webm](https://github.com/Drew246/capstone-project-1/assets/102699958/05be3929-828a-4945-b0d9-7b841a68f2c6)

- The Behance Design
![image](https://github.com/Drew246/capstone-project-1/assets/102699958/6580863a-9b25-475d-85a9-b263624108e9)
![image](https://github.com/Drew246/capstone-project-1/assets/102699958/bfaf4704-09b2-47d6-981a-47e8da08da7e)
![image](https://github.com/Drew246/capstone-project-1/assets/102699958/7e38e3a3-d0c3-4f6b-8eb4-0dbda2ca8645)

Reply:
- [Optional ] A suggested by the previous reviewer mentioned your mobile view for the About page doesn't look as it's supposed to. The entire page looks quite disorganized so kindly fix this and style the elements properly while ensuring the recommended spacing between them .
- Your Design for About page mobile screen size 
 
![test 1](https://github.com/Drew246/capstone-project-1/assets/93598189/3baf6314-ea4f-476d-b520-c0e0cd10ae7f)